modules/conflict_resolver/README.md

```diff
@@ -0,0 +1,68 @@
+# Conflict Resolver
+
+## Purpose
+
+The conflict resolver is intended to allow the resolution of conflicts
+(different data entry for a given field between the "single data
+entry" and "double data entry" variations) for instruments which
+have double data entry enabled.
+
+## Intended Users
+
+The conflict resolver is usually used by data entry staff to resolve
+double data entry conflicts.
+
+Some studies may have protocols which involve supervisors or
+committees deciding how to properly resolve conflicts. For example,
+studies may adopt different policies regarding the personnel that
+should be responsible for conflict resolution (which could be the
+same data entry staff that performed the double data entry or a
+different data entry staff). Such policies are not enforced by
+LORIS.
+
+## Scope
+
+The module only provides data entry conflict resolution and triggers
+the recalculation of fields which are dependent on the data that's
+been changed (ie. score fields, age, etc). It does not do the initial
+data entry on the instruments themselves, only the resolution.
+
+It is up to any given study's internal policy to determine if
+resolution should be done by the same person or a different person
+than the person who did the initial data entry and not enforced by
+LORIS.
+
+## Permissions
+
+Accessing the module requires the `conflict_resolver` permission.
+
+A user should only see conflicts at the sites at which the user is
+a member.
+
+## Configurations
+
+The `DoubleDataEntryInstruments` setting determines whether or not
+an instrument gets flagged for conflicts. This is not directly used
+by the module, but the base instrument class uses this to determine
+whether or not to populate the conflicts table upon data entry
+completion.
+
+## Interactions with LORIS
+
+The conflict resolver module determines which fields are in conflict
+by looking into the `conflicts_unresolved` table, which instruments
+must populate upon the "Data Entry" flag being set to Complete.
```

Comment:
I find this first sentence a bit hard to read for some reason. What about rephrase a bit like this?
... into the `conflicts_unresolved` table, as well as which instruments must populate...

Reply:
but that changes the meaning of the sentence in a way that's inaccurate.

The instruments must populate the table, the conflict resolver doesn't look into them.

Reply:
@driusan haha, ok, then I don't understand the sentence. My bad. Don't know if there is a way to make it clearer or just discard my comment

Reply:
@PapillonMcGill I think that would also have the wrong meaning

Reply:
What about:

```
The conflict resolver module determines which fields are in conflict
by looking into the `conflicts_unresolved` table, which instruments
must populate themselves.
```

?

Reply:
better!

Reply:
Better!!! I get it now :). 👍